src/components/NavBar/NavBar.tsx

```diff
@@ -38,14 +39,16 @@ export default function NavBar() {
             <ul className="border-r border-gray-700">
               <li className="flex items-center gap-8 last:mr-5">
                 {navLinks.map((link) => {
+                  const isActive = (pathname === "/" && link.href === "/") ||
```

Comment:
Made relative link to match the Nav link active style: e.g /blog/title will match /blog